src/EFCore/Diagnostics/PropertyValueEventData.cs

```diff
@@ -43,6 +43,11 @@ public class PropertyValueEventData : PropertyEventData
         /// </summary>
         public virtual EntityEntry EntityEntry { get; }
 
+        /// <summary>
+        ///     The property.
+        /// </summary>
+        new public virtual IProperty Property => (IProperty)base.Property;
```

Comment:
```suggestion
        public new virtual IProperty Property => (IProperty)base.Property;
```

nit: Rider likes sorted modifiers, in case we care... Elsewhere as well...

Reply:
editorconfig has rule for it. Code cleanup should fix this.